### ISSUE TITLE
Failure on Kubuntu 20.04

### ISSUE THREAD

**Issue by PRoots** [rdkr/ubuntu-lvm-on-luks#4]
Hi I have just run the script (sudo) which was fine right until the end when I get
update-initramfs: Generating /boot/init.....-generic
Error 24 : Write error : cannot write compressed block
E: mkinitfamfs failure cpio 141 lz4 -9 -l 24

Any idea what I can do to correct this?
I have checked and there is a full looking file system mounted on /mnt including a /boot
Thanks
Peter

**Comment by PRoots** [comment]
I forgot to mention /boot was not on lvm and I used swap root and home on lvm all of which created successfully
Just tried the install again but this time tried the script after a 'sudo su' to see if it made a difference (it didn't)

**Comment by rdkr** [comment]
hey @PRoots,

i've not come across this before, but as it seems to be going wrong writing the initial ram disk to the boot partition.

how did you create /boot / was it a fresh partition / how large is it / can you write anything to it? i'm not sure of a specific required size, but 1GB should be sufficient i think

**Comment by PRoots** [comment]
Hi, thanks for getting back to me
I created the boot partition of 300Mb - formatted during the install. After install shows as 84% used. readable and writable if I elevate permissions to root - there is 40Mb free space but on checking the initramfs files that exist are twice that so perhaps it is running out of space to write the new ones before removing the old. I picked 300Mb as I read somewhere that 200 was enough but use 300 to be safe. Wrong I guess :-). A machine with a fully encrypted setup (not dual boot) picked by default 730Mb so that is probably a good choice. Running install again now
